### PR TITLE
Improved test coverage in 5 places for production code

### PR DESCRIPTION
**Pull request by mtr-d3v**
This PR fixes issue #629 by Improving test coverage in 5 places for production code

----
📚 Documentation preview 📚: https://icalendar--696.org.readthedocs.build/

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Thanks!

While looking through it: you found mistakes in the code by improving the coverage. 🥳

could you correct that too or create an issue for it?

vCalAddress('b'value'') repr should be vCalAddress('value')

and vBinary repr probably
vBinary(b'value')

My guess is that changing python 2 to 3 did this.

**Comment by mtr-d3v** [comment]
Thanks for reviewing my PR. I have corrected them.

**Comment by niccokunzmann** [comment]
Could you merge master?

File: src/icalendar/tests/prop/test_vBinary.py [hunk 1]
```diff
@@ -27,3 +29,13 @@ def test_long_data():
     txt_ical = b'YWFh' * 33
     assert (vBinary(txt).to_ical() == txt_ical)
     assert (vBinary.from_ical(txt_ical) == txt)
+    
+def test_repr():
+    instance = vBinary("value")
+    assert repr(instance) == "vBinary('b'value'')"
```
Comment (niccokunzmann) [inline src/icalendar/tests/prop/test_vBinary.py]:
```suggestion
    assert repr(instance) == "vBinary(b'value')"
```

File: src/icalendar/tests/prop/test_vCalAddress.py [hunk 2]
```diff
@@ -17,3 +17,8 @@ def test_params():
 
 def test_from_ical():
     assert vCalAddress.from_ical(txt) == 'MAILTO:maxm@mxm.dk'
+    
+
+def test_repr():
+    instance = vCalAddress("value")
+    assert repr(instance) == "vCalAddress('b'value'')"
```
Comment (niccokunzmann) [inline src/icalendar/tests/prop/test_vCalAddress.py]:
```suggestion
    assert repr(instance) == "vCalAddress('value')"
```

**Review by niccokunzmann — APPROVED** [review]
Thanks, I made the changes clearer that I would like to see. Do you think this can be implemented in this pull request?

It is strictly not what the issue said, so I could merge it and create a first-timer issue instead.

What do you think?

**Comment by mtr-d3v** [comment]
I have fixed the issue. Now you can check.

**Comment by niccokunzmann** [comment]
@mtr-d3v Thanks! I just merged master so there is no conflict. It seems that the tests are updated and fail.

Could you [install tox and run the tests in your local setup?](https://icalendar.readthedocs.io/en/latest/install.html#install-tox)

I am wondering why not all the tests were running before ...

**Comment by niccokunzmann** [comment]
Thank you very much!

**Pull request merged**